res/css/views/right_panel/_RoomSummaryCard.scss

```diff
@@ -174,6 +176,20 @@ limitations under the License.
                     mask-image: url('$(res)/img/element-icons/room/pin-upright.svg');
                 }
             }
+            .mx_RoomSummaryCard_app_maximise {
+                right: 48px;
+
+                &::before {
+                    mask-image: url('$(res)/img/feather-customised/maximise.svg');
+                }
+            }
+            .mx_RoomSummaryCard_app_minimise {
+                right: 48px;
+                &::before {
+                    background-color: $accent-color;
```

Comment:
Looks like this line is causing build failures on CI... I believe it's because some colour variables were recently renamed. You might need to merge in develop and tweak this line to match the new name.

Reply:
Does this mean, its already doing a build with merging to develop?
It does compile with just this branch on my machine.

Reply:
No, CI is compiling your branch without merging anything, but for the other layers (element-web, matrix-js-sdk), it will default to develop, so I believe that's where the mix-up is coming from. I assume a newer element-web with this older matrix-react-sdk was causing the build failure.

If you were working on a PR across layers and use the same branch name on each repo, then it pulls your matching branches on the other layers, but by default develop is used.

Reply:
You may need to pull in the latest develop changes in the element-web layer locally to see the same failure that CI is seeing.